applications/ContactStructuralMechanicsApplication/custom_includes/point_item.h

```diff
@@ -142,9 +142,19 @@ class PointItem: public Point<3>
      * @return Coordinates: The coordinates of the item
      */
 
-    void UpdatePoint()
-    {
-        this->Coordinates() = mpOriginCond->GetGeometry().Center().Coordinates();
+    void UpdatePoint(const double& DeltaTime)
+    {        
+        Point<3> center;
+        if (mpOriginCond->GetGeometry()[0].SolutionStepsDataHas(VELOCITY_X) == true)
+        {
+            Point<3> center = ContactUtilities::GetHalfJumpCenter(mpOriginCond->GetGeometry(), DeltaTime); // NOTE: Center in half delta time
```

Comment:
redefining the center.

Reply:
What do you mean?

Reply:
line 147 and line 150 are defining `center` as variable